massdriver-application-gcp-vm/autoscale.tf

```diff
@@ -0,0 +1,18 @@
+resource "google_compute_autoscaler" "main" {
+  for_each = toset(data.google_compute_zones.available.names)
+  provider = google-beta
+
+  name   = module.application.params.md_metadata.name_prefix
+  zone   = each.value
+  target = google_compute_instance_group_manager.main[each.value].id
+
+  autoscaling_policy {
+    max_replicas    = 5
```

Comment:
I've given similar feedback on other scaling configs. IMHO they should not be hard coded, but user configurable with reasonable defaults.

Is there no valid scale to zero vm app case?
Do we think scaling beyond 5 will never happen?

especially in the module I think better to be more flexible.

Reply:
Thanks for making this comment again, I appreciate it. I'm trying to balance iterating and configuration to try and get things in front of customers faster. Changing this in the future would be easy, but the cycles def annoying and time-consuming. Max-replicas should be part of the massdriver.yaml first-pass. 👌 

For 0, I was curious. However... the health check keeps it alive. If we want to support 0 we'd have to remove that, or create the HC when scale is > 0. I haven't removed a HC to test going from 0 to 1 so I don't know if it's automatic or if we have to keep a minimum for API-like services.